flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -904,7 +904,10 @@
         "module-totem-source-git-no-commit-or-tag": "This is the development version of Totem, tracking upstream"
     },
     "org.fcitx.Fcitx5": {
-        "finish-args-arbitrary-dbus-access": "IDEs tend to require direct dbus access"
+        "finish-args-arbitrary-dbus-access": "IDEs tend to require direct dbus access",
+        "finish-args-contains-both-x11-and-wayland": "Input method daemon may communicate with both XWayland/X11/Wayland applications at the same time",
```

Comment:
this one is a warning. so you don't need an exception

Reply:
Done